### DISCUSSION TITLE
Hide a table's fields when adding records via relational fields.

### DISCUSSION THREAD

**Discussion by mmmaxxxe** [directus/directus#13984] [Archive]
Hi!
don't know if it has been asked or if the solution is still there (latest directus cloud).

Collection A
- name
- surname
- other (O2M, related to collection B) 
- one more other (O2M related to collection C)
- even one more (M2M related to collection D)

Collection B

- attribute
- more attribute
- other (M2O related to Collection A -> other)

If i try to add a record to Collection B -> "other" via the M2O interface to the collection A i see the whole Collection A (all fields). Would it be possible to see only:
-name
-surname 

maybe?

Thanks to everybody wanting to help.

**Comment by rijkvanzanten** [comment]
> If i try to add a record to Collection B -> "other" via the M2O interface to the collection A i see the whole Collection A (all fields). Would it be possible to see only:

It currently uses the layout configured for Collection A. If the user has that configured to show all fields, all fields will be visible. You could set a default preset for your system from Settings to control what that layout looks like for users who haven't configured anything themselves yet

**Reply by mmmaxxxe** [reply]
Hi Rijk,
thanks for your reply
it's not actually a question of current user, but of a event calling the form.
I need the whole form/collection from collection A normally, just part of it in case it is called via the M2O

Thanks
